app/src/main/java/com/mapbox/maps/testapp/examples/annotation/PointAnnotationActivity.kt

```diff
@@ -52,17 +54,33 @@ class PointAnnotationActivity : AppCompatActivity() {
 
         addClickListener(
           OnPointAnnotationClickListener {
-            Toast.makeText(this@PointAnnotationActivity, "Click: ${it.id}", Toast.LENGTH_SHORT)
+            Toast.makeText(this@PointAnnotationActivity, "Click1: ${it.id}", Toast.LENGTH_SHORT)
```

Comment:
Perhaps just `Click` and not `Click1`?

Reply:
Oh, I see, there's a second one as well. If it's possible logically perhaps better use some names and 1 or 2. But I'm OK with 1 or 2 although sounds a bit verbose.